### PR TITLE
Adding heatmap to documentation index for distro

### PR DESCRIPTION
**Pull request by eybee**

### CODE REVIEW

**Comment by vrabaud** [comment]
the branch for doc does not exist

**Comment by eybee** [comment]
Updated!

File: jade/distribution.yaml [hunk 1]
```diff
@@ -929,6 +929,21 @@ repositories:
...
     status: maintained
+  heatmap:
+    doc:
+      type: git
+      url: https://github.com/eybee/heatmap.git
+      version: jade-devel
+    release:
+      tags:
+        release: release/jade/{package}/{version}
+      url: https://github.com/eybee/heatmap-release
```
Comment (dirk-thomas) [inline jade/distribution.yaml]: The url must end with `.git`.

**Comment by eybee** [comment]
Thanks, updated!

File: jade/distribution.yaml [hunk 2]
```diff
@@ -929,6 +929,21 @@ repositories:
...
     status: maintained
+  heatmap:
+    doc:
+      type: git
+      url: https://github.com/eybee/heatmap.git
+      version: jade-devel
+    release:
+      tags:
+        release: release/jade/{package}/{version}
+      url: https://github.com/eybee/heatmap-release.git
```
Comment (dirk-thomas) [inline jade/distribution.yaml]: This is not a release repository. You might want to look into the first bloom tutorial (http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease) how to release ROS packages.

**Comment by wjwwood** [comment]
@eybee when you address a comment on a pull request you need to make a comment too. We don't get notified when you push a new commit. Thanks!

**Pull request merged**